modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -56,58 +56,59 @@ Below you can see what the core runtime’s hierarchy looks like:
 
 image::error-hierarchy.png[Error Hierarchy]
 
-All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for specification of unclear errors in the future, without changing the existing behavior of the app.
+All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow specification of unclear errors in the future, without changing the existing behavior of the app.
```

Comment:
"to allow specification" -> "to allow for the specification of" 
Or that sentence could actually be broken up and clarified perhaps. Seems to be trying to say too much in one sentence.